core/src/cartridge.rs

```diff
@@ -188,13 +188,26 @@ impl PartialOrd for RomSize {
     }
 }
 
-/// Size of a cartridge's RAM. Specified in KiB.
+/// Size of a cartridge's RAM. Specified in KiB. Each RAM bank can hold 8KiB.
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
 pub enum RamSize {
     None,
+
+    /// Only the first quarter of the RAM address space (0xA000 - 0xA7FF) is
+    /// valid RAM.
     Kb2,
+
+    /// One bank, full address space used.
     Kb8,
-    Kb32, // 4 banks of 8KBytes each
+
+    /// 4 banks.
+    Kb32,
+
+    /// 16 banks.
+    Kb128,
```

Comment:
According to [\[1\]](http://gbdev.gg8.se/wiki/articles/MBC5) this size does not exist. This should be replaced by 256Kb and 1Mb.

- \[1\]: http://gbdev.gg8.se/wiki/articles/MBC5

Reply:
The sizes on that page are given in K**bit** and M**bit**. Divide by 8 to get 32 KiB and 128 KiB.

Reply:
My fault, I interpreted your comment on the enum wrong `KiB` means of course Byte 🤦‍♂️